src/NzbDrone.Core/Download/Clients/Transmission/TransmissionBase.cs

```diff
@@ -73,8 +73,21 @@ protected virtual OsPath GetOutputPath(OsPath outputPath, TransmissionTorrent to
             return outputPath + torrent.Name.Replace(":", "_");
         }
 
-        protected string GetDownloadDirectory()
+        protected string GetDownloadDirectory(string category)
         {
+            if (category.IsNotNullOrWhiteSpace())
```

Comment:
this feels a bit clunky but not sure of a better way to factor it without extra calls to `_proxy.GetConfig()`